drivers/storage/s3fs/utils/utils.go

```diff
@@ -0,0 +1,25 @@
+// +build !libstorage_storage_driver libstorage_storage_driver_s3fs
+
+package utils
+
+import (
+	"os"
+
+	gofig "github.com/akutz/gofig/types"
+
+	"github.com/codedellemc/libstorage/api/types"
+	"github.com/codedellemc/libstorage/drivers/storage/s3fs"
+)
+
+// InstanceID returns the instance ID for the local host.
+func InstanceID(
+	ctx types.Context, config gofig.Config) (*types.InstanceID, error) {
+
+	var hostName string
+	if config == nil {
+		hostName = config.GetString(s3fs.ConfigS3FSHostName)
+	} else {
+		hostName, _ = os.Hostname()
```

Comment:
It wasn't until I looked at this code that it dawned on me that this driver is useful outside of AWS. Of course it is, but I just hadn't thought about it. Might be something worth pointing out in the docs, because it is really useful?

Would there be any merit it trying to make sure there is a more uniqueness to the IID, by appending a MAC address or anything? Probably not, but I think once you jump out of the cloud provider, you are much more likely to run into hostname collisions.